### DISCUSSION TITLE
你们的PikPak还能正常挂载吗？

### DISCUSSION THREAD

**Discussion by emodiyu** [AlistGo/alist#7083] [General]
我的在一周前莫名不能挂载了，账号都是正常的
提示信息：failed get objs: failed to list objs: oauth2: "invalid_argument" "currently not supported"

**Comment by ririye** [comment]
一样的不正常情况

**Comment by Torjoy** [comment]
不行，据说beta版可以了，但是我是装的docker版的，没找到beta版更新，有消息了踢我一下谢谢

**Reply by ririye** [reply]
https://github.com/alist-org/alist/releases/tag/beta
这里下载
